### PR TITLE
Add admin upload api

### PR DESCRIPTION
**Pull request by zozovp**
https://github.com/DesignGarage1/SimplePrintsiOS/issues/3633

### CODE REVIEW

**Comment by zozovp** [comment]
@NovaDev94 please have a check when you are free mate.

File: routers/uploadToS3.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+var uploadToS3 = function(fileStream, next) {
+  var s3Client = knox.createClient({
+        secure: false,
+        key: config.AWS_KEY,
+        secret: config.AWS_SECRET,
+        bucket: config.AWS_BUCKET,
+      });
+  var id = uuid.v4();
+  var timestamp = Date.now();
+  var s3Key = id + '/original/' + timestamp + '.jpg';
```
Comment (NovaDev94) [inline routers/uploadToS3.js]: This just doesn't follow our convention of base url.

Maybe just `s3Key = id + '. jpg'` is fine.

**Comment by NovaDev94** [comment]
Beside of my last comment, this PR LGTM

File: routers/uploadToS3.js (hunk 1 above)
Comment (zozovp) [inline routers/uploadToS3.js]: thanks mate

**Pull request merged**